lib/snippets/publish-lerna-independent-packages

```diff
@@ -28,10 +28,12 @@ const packages = PackageUtilities.topologicallyBatchPackages(unsortedPackages, t
   .reduce((acc, packageGroup) => [...acc, ...packageGroup], [])
   .filter(({name}) => taggedPackages.includes(name));
 
-packages.forEach(({name, version}) => {
-  const command = `node_modules/.bin/lerna publish --yes --skip-npm=false --skip-git --force-publish=${name} --repo-version=${version} --scope=${name} --npm-tag=${npmTag(
-    version,
-  )}`;
+packages.forEach(({name, version, isPrivate}) => {
+  const access = isPrivate ? 'restricted' : 'public';
```

Comment:
where is `isPrivate` being set?

Reply:
It comes from the `lerna` utility that we're using. That said, this is going to change in `lerna` version `3` 🤔: https://github.com/lerna/lerna/commit/75e3a4aa3ad879e8f26695b85c52dc931e1f4eb8

Reply:
`packages` is an array of package descriptions generated by lerna's `PackageUtilities.topologicallyBatchPackages`

Reply:
maybe we can just drop the `access` flag entirely and rely on the `publishConfig`?

Reply:
I think the best approach is to just let lerna tell us using the current api and worry about upgrading when we upgrade. That said, I'm going to push a commit in a sec that pins the lerna version as part of the `lerna_discovery` script.

Reply:
💯